css/reset.css

```
@@ -60,6 +46,7 @@ button,
 input,
 select,
 textarea {
+  font-family: Pretendard;
```

💊 제안
지금의 경우는 html에 적용된 fontFamliy를 상속받는게 더 중복없고 의도가 잘 드러나는 것 같아서 아래처럼 예약어를 사용하시기를 추천드려요!

```suggestion
  font-family: inherit;
```